python_modules/dagster/dagster/_core/definitions/declarative_scheduling/updated_since_cron_condition.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+
+from dagster._utils.schedules import reverse_cron_string_iterator
+
+from .asset_condition import AssetCondition, AssetConditionResult
+from .scheduling_condition_evaluation_context import SchedulingConditionEvaluationContext
+
+
+class UpdatedSinceCronCondition(AssetCondition):
```

Comment:
Given that this is new code, seems like it should be written against `SchedulingCondition`?

Reply:
yep -- updated upstack